VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participants-panel/participants-panel.component.spec.ts

```diff
@@ -555,6 +555,20 @@ describe('ParticipantsPanelComponent', () => {
         component.toggleMuteParticipant(pat);
         expect(videocallService.muteParticipant).toHaveBeenCalledWith(pat.pexipId, false, component.conferenceId, pat.id);
     });
+    describe('handleParticipantMediaStatusChange', () => {
+        it('should call updateParticipant for a linked participant witha hearing role interpreter', () => {
+            // Arrange
+            const mediaStatus = new ParticipantMediaStatus(true, false);
+            const interpreter = component.participants.filter(x => x instanceof LinkedParticipantPanelModel)[0];
```

Comment:
With the current data this will get one with PanelMembers.

Reply:
it has been removed